src/lib.rs

```diff
@@ -197,32 +297,56 @@ macro_rules! map_enum {
 pub extern "C" fn wgpuCreateInstance(
     _descriptor: *const native::WGPUInstanceDescriptor,
 ) -> native::WGPUInstance {
-    // Rationale: See https://github.com/gfx-rs/wgpu-native/issues/116
-    // Because WGPUInstance is an opaque type this library controls and does not define the contents of, this is safe.
-    8 as native::WGPUInstance
+    let context = Context::new(
+        "wgpu",
+        wgc::hub::IdentityManagerFactory,
+        wgt::Backends::PRIMARY,
```

Comment:
we should take backend via extras, same as `wgpuRequestAdapter`

Reply:
Added backend flags so you can indicate which backends are enabled by the hub